src/iconid.rs

```diff
@@ -40,14 +51,31 @@ impl IconIdentifier {
                 .map_err(IconResolutionError::ReadError)?
                 .map_codepoint(*cp)
                 .ok_or(IconResolutionError::NoCmapEntry(*cp)),
-            IconIdentifier::Name(name) => resolve_ligature(font, name.as_str()),
+            IconIdentifier::Name(name) => font.resolve_ligature(name.as_str()),
         }?;
 
         apply_location_based_substitution(font, location, gid)
             .map_err(IconResolutionError::ReadError)
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub struct Icon {
```

Comment:
Why do we need a public Icon struct and not merely to enumerate the name strings?